pkg/web/save.go

```diff
@@ -37,6 +37,30 @@ func Save(
 		} // else not cached...
 	}
 
+	if err := SaveNoCheck(url, download, binFile); err != nil {
+		fmt.Printf("download problem:%s", err)
+	}
+
+	// Now the file is updated - update the checksum...
+	hash, err := hashcache.UpdateCache(download)
+	if err != nil {
+		return err
+	}
+	if !hashcache.IsCachedMatch(download, sha256) {
+		log.Printf("invalid checksum (%s) for %s, expecting %q", sha256, download, hash)
+		return errors.Errorf("invalid checksum for %s", download)
+	} else {
```

Comment:
we could drop the else .. as the return negates it ..

Reply:
totes - nice spot.